rmgpy/data/kinetics/family.py

```diff
@@ -1605,7 +1606,8 @@ def __generateReactions(self, reactants, products=None, forward=True, prod_reson
                             Default is True, labels will be cleared
 
         Returns:
-            List of all reactions with the specified reactants and products within this family.
+            List of all reactions containing Species objects with the
```

Comment:
I think `family.generateReactions` and `__generateReactions` returns reactions containing `Molecule` objects instead of `Species` objects. `generate_reactions_from_families` indeed returns  reactions containing `Species` objects because it calls `find_degenerate_reactions` which calls `ensure_species_in_reaction`. Can you double-check that? also if that's true, is it good to have this Molecule-Species conversion happen so implicitly?

Reply:
Thanks for catching that! They do return reactions with `Molecule` objects. I don't think there's much we can do about the conversion though. Let me know if you have any ideas.